locales/jp.yml

```diff
@@ -1,11 +1,11 @@
 ---
 jp:
-  navigation:
+  global_navigation:
     support: サポート
-    contribute: Contribute
+    community: Community
     docs: Docs
     logo: Middleman
-    search_placeholder: ドキュメント検索
```

Comment:
Looks like it was originally added long ago in https://github.com/middleman/middleman-guides/commit/c339021ed772f1c4ee369f3f5e773620853ada1f#diff-98e97cfa6efa32a2e18600bef3e4bc27R7, but wasn’t used when the search functionality changed.